nil pointer dereference when running go-bootstrap on linux/amd64

### Steps to Reproduce

On linux/amd64, run these commands:

```
go get github.com/go-bootstrap/go-bootstrap
$GOPATH/bin/go-bootstrap -dir github.com/xxxx/yyyy
```

### Observed Behavior

```
2015/07/26 00:09:47 Creating /home/aev/go/src/github.com/xxxx/yyyy...
2015/07/26 00:09:47 Copying a blank project to /home/aev/go/src/github.com/xxxx/yyyy...
2015/07/26 00:09:47 Replacing placeholder variables on xxxx/yyyy...
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x403709]

goroutine 1 [running]:
main.main()
	/home/aev/go/src/github.com/go-bootstrap/go-bootstrap/main.go:91 +0x2b09
```

### Expected Behavior

`go-bootstrap` should work on linux/amd64.

### Commentary

`currentUser` on line 91 of main.go is nil. Replacing lines 59 and 60 of main.go with those below produces the error "user: Current not implemented on linux/amd64".

```go
  currentUser, err := user.Current()
  helpers.ExitOnError(err, "")
```

See also #1.

--- a/helpers/helpers.go
+++ b/helpers/helpers.go
@@ -119,8 +119,7 @@ func DefaultPGDSN(dbName string) string {
 	hostPortSeparator := ":"
 
 	if pguser == "" {
-		u, _ := user.Current()
-		pguser = u.Username
+		pguser = GetCurrentUser()
 	}
 
 	isUnixDomainSocket := strings.HasPrefix(pghost, "/")
@@ -152,5 +151,22 @@ func DefaultPGDSN(dbName string) string {
 func ExitOnError(err error, msg string) {
 	if err != nil {
 		log.Fatalf("%s\n%s", msg, err.Error())
+	}
+}
+
+// GetCurrentUser returns the username of the current user.
+func GetCurrentUser() string {
+	currentUser, err := user.Current()
+
+	if err == nil {
+		return currentUser.Username
+	} else {
+		username := os.Getenv("USERNAME")
+
+		if username == "" {
+			log.Fatalln("Cannot determine current user's username")
+		}
+
+		return username
 	}
 }

--- a/main.go
+++ b/main.go
@@ -6,7 +6,6 @@ import (
 	"log"
 	"os"
 	"os/exec"
-	"os/user"
 	"path/filepath"
 	"strings"
 
@@ -56,20 +55,6 @@ func main() {
 	projectName := dirChunks[len(dirChunks)-1]
 	dbName := projectName
 	testDbName := projectName + "-test"
-
-	var userName string
-	currentUser, err := user.Current()
-
-	if err == nil {
-		userName = currentUser.Username
-	} else {
-		userName = os.Getenv("USERNAME")
-
-		if userName == "" {
-			log.Fatalln("Cannot determine current user's username")
-		}
-	}
-
 	blankDir, err := helpers.GetBlankDir()
 	helpers.ExitOnError(err, "")
 
@@ -100,7 +85,7 @@ func main() {
 	replacers["$GO_BOOTSTRAP_REPO_USER"] = repoUser
 	replacers["$GO_BOOTSTRAP_PROJECT_NAME"] = projectName
 	replacers["$GO_BOOTSTRAP_COOKIE_SECRET"] = helpers.RandString(16)
-	replacers["$GO_BOOTSTRAP_CURRENT_USER"] = userName
+	replacers["$GO_BOOTSTRAP_CURRENT_USER"] = helpers.GetCurrentUser()
 	replacers["$GO_BOOTSTRAP_PG_DSN"] = helpers.DefaultPGDSN(dbName)
 	replacers["$GO_BOOTSTRAP_ESCAPED_PG_DSN"] = helpers.BashEscape(helpers.DefaultPGDSN(dbName))
 	replacers["$GO_BOOTSTRAP_PG_TEST_DSN"] = helpers.DefaultPGDSN(testDbName)

--- a/main.go
+++ b/main.go
@@ -56,7 +56,19 @@ func main() {
 	projectName := dirChunks[len(dirChunks)-1]
 	dbName := projectName
 	testDbName := projectName + "-test"
-	currentUser, _ := user.Current()
+
+	var userName string
+	currentUser, err := user.Current()
+
+	if err == nil {
+		userName = currentUser.Username
+	} else {
+		userName = os.Getenv("USERNAME")
+
+		if userName == "" {
+			log.Fatalln("Cannot determine current user's username")
+		}
+	}
 
 	blankDir, err := helpers.GetBlankDir()
 	helpers.ExitOnError(err, "")
@@ -88,7 +100,7 @@ func main() {
 	replacers["$GO_BOOTSTRAP_REPO_USER"] = repoUser
 	replacers["$GO_BOOTSTRAP_PROJECT_NAME"] = projectName
 	replacers["$GO_BOOTSTRAP_COOKIE_SECRET"] = helpers.RandString(16)
-	replacers["$GO_BOOTSTRAP_CURRENT_USER"] = currentUser.Username
+	replacers["$GO_BOOTSTRAP_CURRENT_USER"] = userName
 	replacers["$GO_BOOTSTRAP_PG_DSN"] = helpers.DefaultPGDSN(dbName)
 	replacers["$GO_BOOTSTRAP_ESCAPED_PG_DSN"] = helpers.BashEscape(helpers.DefaultPGDSN(dbName))
 	replacers["$GO_BOOTSTRAP_PG_TEST_DSN"] = helpers.DefaultPGDSN(testDbName)

